rmw_implementation/test/test_functions.cpp

```diff
@@ -17,7 +17,6 @@
 #include <memory>
 
 #include "rcutils/env.h"
-#include "rcutils/get_env.h"
```

Comment:
Unfortunately, we can't do this here because `rcutils_get_env` is used below (at least, we can't do this prior to https://github.com/ros2/rcutils/pull/340 being merged).  So either this has to remain part of the larger series to move `rcutils_get_env` into env.h, or we have to switch the test code to use `rcpputils::get_env`.  Either is fine with me.

Reply:
`rcutils/env.h` includes `rcutils/get_env.h`, though: https://github.com/ros2/rcutils/blob/ca42bc9f588e11ad40f5879dcb9a364b435d2cba/include/rcutils/env.h#L31

Reply:
> `rcutils/env.h` includes `rcutils/get_env.h`, though: https://github.com/ros2/rcutils/blob/ca42bc9f588e11ad40f5879dcb9a364b435d2cba/include/rcutils/env.h#L31

Ah!  I totally missed that.  Thanks for the heads up.  I'll approve and run CI on this, then.